get_genotype_vcf.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+# pylint: disable=import-error
+
+"""
+This script will
+
+- extract common variants
+- export as VCF (one per chrom)
+
+this will be used as input for the
+SAIGE-QTL association pipeline.
+
+
+To run:
+
+analysis-runner \
+    --description "get common variant VCF" \
+    --dataset "bioheart" \
+    --access-level "test" \
+    --output-dir "saige-qtl/input_files/genotypes/" \
+    python3 get_genotype_vcf.py --vds-name vds/1-0.vds --chromosomes chr22
+
+"""
+
+# python modules
+from cpg_utils.config import get_config
+from cpg_utils.hail_batch import dataset_path, get_batch, init_batch, output_path
+
+import click
+import hail as hl
+from hail.methods import export_vcf
+
+
+BCFTOOLS_IMAGE = get_config()['images']['bcftools']
+
+# inputs:
+@click.option('--vds-name', help='e.g., vds/1-0.vds')
+@click.option('--chromosomes', help='e.g., chr22')
```

Comment:
```suggestion
@click.argument('--chromosomes', help='e.g., chr22 chrX ', nargs=-1)
```

alternative is to switch to click arguments. This would make your command line call something like

```
python3 get_genotype_vcf.py --vds-name vds/1-0.vds chr1 chr2 chr22
```

(no need to use `--chromosomes` at all)

Reply:
what does `nargs=-1 `do?